Subject: disable autosuspend for usb printers
Patch-mainline: no

This patch disables autosuspend for USB printers,
as it is unlikely that a computer connected to a
printer operates on battery and some printers are
incompatible with autosuspension

Signed-off-by: Oliver Neukum <oneukum@suse.de>
---
---
 drivers/usb/class/usblp.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/drivers/usb/class/usblp.c	2007-07-08 19:32:17.000000000 -0400
+++ b/drivers/usb/class/usblp.c	2007-08-27 14:01:23.000000000 -0400
@@ -1246,7 +1246,9 @@ static struct usb_driver usblp_driver = 
 	.suspend =	usblp_suspend,
 	.resume =	usblp_resume,
 	.id_table =	usblp_ids,
-	.supports_autosuspend =	1,
+	/* disabled due to little practical signifocance
+	 * and some broken devices */
+	/* .supports_autosuspend =	1, */
 };
 
 static int __init usblp_init(void)
